src/main/java/frc/robot/Subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,95 @@
+package frc.robot.Subsystems;
+
+
+import java.util.function.DoubleSupplier;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/** A robot arm subsystem that moves with a motion profile. */
+public class ArmSubsystem extends SubsystemBase {
+  private CANSparkMax armMotor;
+  private SparkMaxPIDController m_pidController;
+  private RelativeEncoder m_encoder;
+  public double kP, kI, kD, kIz, kFF, kMaxOutput, kMinOutput;
+
+  public ArmSubsystem() {
+    armMotor = new CANSparkMax(Constants.ARM_MOTOR_ID, MotorType.kBrushless);
+    m_encoder = armMotor.getEncoder();
+
+    armMotor.setIdleMode(IdleMode.kBrake);
+    armMotor.restoreFactoryDefaults();
+    m_pidController = armMotor.getPIDController();
+
+    // PID coefficients
```

Comment:
Ideally we should add all of these to our Constants class. It's OK for now, but I'd like this to be tagged with a TODO comment indicating that we should move these.